walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -36,9 +36,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     long usedMemory = Runtime.getRuntime().totalMemory() - Runtime.getRuntime().freeMemory();
 
     // Convert the server stats to JSON
-    ServerStats serverStats = new ServerStats(startTime, currentTime, maxMemory, usedMemory);
```

Comment:
hmm, did you mean to make this change?

Reply:
It was on accident I don't really know what changed so I didn't know what to do other than add it.

Reply:
git checkout ${commit} {file} will update the file to a version at a particular commit. Then you can git commit.

Try
`git checkout master walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java`

(In this case, master doesn't refer to a branch, but is an alias for the commit at the head of the master branch)

Reply:
solved.... I think